courses/OfferedCourses.md

```diff
@@ -1,13 +1,10 @@
 ---
-id: index
-title: Index
-sidebar_label: Index
+id: Offered Courses
+title: Offered Courses
+sidebar_label: Offered Courses
 slug: /
-hide_table_of_contents: true
+hide_table_of_contents: false
```

Comment:
This doesnt work for all monitor sizes, it only works on larger ones

If you go down to the moment before it goes into tablet mode the table is completely cut off so having it set to `true` is necessary not to mention do we need a table of contents for a table?

Reply:
Oh, I wasn't aware of this. Well in that case we can remove it for now